src/pke/lib/scheme/ckksrns/ckksrns-parametergeneration.cpp

```diff
@@ -316,16 +316,21 @@ void ParameterGenerationCKKSRNS::CompositePrimeModuliGen(std::vector<NativeInteg
                         }
                     }
                     catch (const OpenFHEException& ex) {
-                        OPENFHE_THROW(
-                            "COMPOSITE SCALING next prime sampling error. "
-                            "Try increasing scaling factor (scalingModSize) "
-                            "or decreasing first modulus size (firstModSize) "
-                            "or try another combination of firstModSize and scalingModSize.");
+                        std::string errMsg = "COMPOSITE SCALING next prime sampling error. ";
```

Comment:
Are you sure that you want to override OpenFHEException caught in 4 places in this file? Also, the new error message is very long. I would throw a shorter error message combined with the original error message. And the new error message can be left in the same place commented out (for debugging purposes)